.github/workflows/build.yml

```diff
@@ -53,9 +53,6 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1
 
-      - name: Copy minimal languages config
-        run: cp .github/workflows/languages.toml ./languages.toml
```

Comment:
Sure, we can do that. It just means that a contributor will have to remember to add a language if they want to add tests for it. And it will be a bit confusing because their tests will pass locally for them, but fail on GitHub, and for not obvious reasons. It was failing for me on the C indentation test, but the failure was the wrong number of spaces. It took a while to figure out it was because of the grammar missing.

Reply:
That's a good point, maybe we should include `languages.toml` in the test directory, then conditionally compile using that languages.toml over the one in the root dir if running tests?

Reply:
Hmm, I just tried implementing this, but I just realized, the feature flag would help if we were only running one test command, but if we separate regular unit tests from integration tests into separate phases, then I'm not sure how to detect if we're running unit tests without the integration feature flag.